src/picongpu/include/particles/pusher/particlePusherPhot.hpp

```diff
@@ -33,14 +33,15 @@ namespace picongpu
         {
 
             template<typename T_Efield, typename T_Bfield, typename T_Pos, typename T_Mom, typename T_Mass,
-                     typename T_Charge>
+                     typename T_Charge,  typename T_Weighting>
                     __host__ DINLINE void operator()(
                                                         const T_Bfield ,
                                                         const T_Efield ,
                                                         T_Pos& pos,
                                                         T_Mom& mom,
                                                         const T_Mass ,
-                                                        const T_Charge )
+                                                        const T_Charge ,
```

Comment:
Could you please remove the `space` after the variable name. Please do the some for other pushers. I missed that you add `space`'s in issue #1210